bin/fetch_papertrail_logs.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+#
+# Use to download archived logs from papertrail between two dates
+#
+# Usage
+#  $ ./bin/fetch_papertrail_logs.sh [from] [to] [output_file]
+#
+# Arguments
+#  from - YYYY-MM-DD date string; lower date limit for log retrieval
```

Comment:
@eliascodes does this default to beginning or end of day?

Reply:
AFAIK beginning of day